src/main/java/org/commonwl/view/workflow/QueuedWorkflow.java

```diff
@@ -5,10 +5,12 @@
 import org.commonwl.view.cwl.CWLToolStatus;
 import org.springframework.data.annotation.Id;
 
+import java.util.List;
+
 /**
  * A workflow pending completion of cwltool
  */
-@JsonIgnoreProperties(value = {"id", "tempRepresentation"})
+@JsonIgnoreProperties(value = {"id", "tempRepresentation", "workflowList"})
```

Comment:
but REST API needs to return the list somehow - `300 Multiple Choice` is not a perfect fit, but probably best we've got.